From: Matthew Wilcox <matthew.r.wilcox@intel.com>
Date: Tue, 3 Jun 2014 17:06:26 -0400
Subject: NVMe: Delete NVME_GET_FEAT_TEMP_THRESH
References: bnc#913030,FATE#317455
Patch-Mainline: v3.17
Git-commit: de672b9748f78dcbc663e12ea44cb24dc287baf0

This define isn't used, and any code that wanted to use it should use
NVME_FEAT_TEMP_THRESH instead.

Signed-off-by: Matthew Wilcox <matthew.r.wilcox@intel.com>
Acked-by: Hannes Reinecke <hare@suse.de>
---
 drivers/block/nvme-scsi.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/block/nvme-scsi.c b/drivers/block/nvme-scsi.c
index 7ec61c4..13403ee 100644
--- a/drivers/block/nvme-scsi.c
+++ b/drivers/block/nvme-scsi.c
@@ -240,7 +240,6 @@ static int sg_version_num = 30534;	/* 2 digits for each component */
 
 /* NVMe Namespace and Command Defines */
 #define NVME_GET_SMART_LOG_PAGE				0x02
-#define NVME_GET_FEAT_TEMP_THRESH			0x04
 #define BYTES_TO_DWORDS					4
 #define NVME_MAX_FIRMWARE_SLOT				7
 
-- 
1.8.5.2

